config/environments/production.rb

```diff
@@ -1,7 +1,8 @@
 require 'yaml'
+require "active_support/core_ext/integer/time"
 Rails.application.configure do
   deliver_method = ENV.fetch('MAIL_DELIVERY_METHOD') { 'smtp' }.to_sym
-  slack_config = Rails.application.config_for(:exception_notifier)[:slack]
+  slack_config = Rails.application.config_for(:exception_notifier)&.fetch(:slack, nil)
```

Comment:
Is this even being used?

Reply:
The config doesn't appear to be used here, but it is in NotifierConfig.  I'll pull it and see what shakes out.